redbeams/src/main/java/com/sequenceiq/redbeams/service/crn/CrnService.java

```diff
@@ -0,0 +1,24 @@
+package com.sequenceiq.redbeams.service.crn;
+
+import java.util.UUID;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.auth.altus.Crn;
+
+@Service
+public class CrnService {
```

Comment:
Maybe cloud-common / common / core-api is a better home for this. It seems very generally useful.